templates/test/overview.html.ep

```diff
@@ -11,10 +11,10 @@
     <div id="summary" class="panel <%= ($aggregated->{failed} + $aggregated->{incomplete}) ? 'panel-danger' : 'panel-success' %>">
         <div class="panel-heading">
             Overall Summary of
-            % if ($group) {
-                <b><%= link_to $group->name => url_for('group_overview', groupid => $group->id) %></b>
-            % } else
-            % {
+            % if (@$groups) {
+                <b><%= b join(', ', map { link_to $_->name => url_for('group_overview', groupid => $_->id) } @$groups) %></b>
```

Comment:
https://github.com/os-autoinst/openQA/pull/1040/files#diff-04c360cd1e676d2393bf877809ebf2c4R147 is the location in the test where this is checked. For now what I added is handling of multiple groupids, the specification of no group or groupid at all can be called "unspecified".